tuf/ngclient/updater.py

```diff
@@ -66,6 +66,7 @@
 from tuf.api.metadata import Root, Snapshot, TargetFile, Targets, Timestamp
 from tuf.ngclient._internal.trusted_metadata_set import TrustedMetadataSet
 from tuf.ngclient.config import EnvelopeType, UpdaterConfig
+from tuf.ngclient.file_lock import lock_file
```

Comment:
```suggestion
from tuf.ngclient._internal.file_lock import lock_file
```

maybe hide this in _internal so we make it absolutely clear these hacks are not API.

Reply:
I will move it.